src/rgw/rgw_sal_motr.cc

```diff
@@ -2201,7 +2290,9 @@ int MotrObject::update_version_entries(const DoutPrefixProvider *dpp)
     ent.flags = rgw_bucket_dir_entry::FLAG_VER;
     string key;
     if (ent.key.instance.empty())
-      key = ent.key.name;
+    {  
```

Comment:
its fine, for single line brackets are not needed

Reply:
sure, done